frontend/taipy-gui/src/components/Taipy/StatusList.tsx

```diff
@@ -95,6 +97,16 @@ const StatusList = (props: StatusListProps) => {
     const [opened, setOpened] = useState(false);
     const [multiple, setMultiple] = useState(false);
     const [anchorEl, setAnchorEl] = useState<HTMLElement | null>(null);
+    const content = useMemo(() => {
+        if (props.customIcon) {
+            try {
+                return props.customIcon.split(';');
+            } catch (e) {
+                console.info(`Error parsing custom icons\n${(e as Error).message || e}`);
```

Comment:
How could this throw an exception except if this is not a string ?
Better test the type and ignore if it is not a string